src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Filters/ShapeStringifyFilter.cs

```diff
@@ -27,7 +21,9 @@ public ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArguments argu
         {
             static async ValueTask<FluidValue> Awaited(Task<IHtmlContent> task)
             {
-                return new HtmlContentValue(await task);
+                using var writer = new StringWriter();
```

Comment:
Not good

Reply:
If the goal to invert the call with the encoder, this is not necessary anymore. It doesn't call it char by char like before

Reply:
It is the `shape_stringify` filter, not the `shape_render`, it is not intended to be written to the buffer, but e.g. to assign a local liquid var. It was already using a `StringWriter` below when the task completed immediately

Here in the `Awaited()` method it was using an `HtmlContentValue` because i assume a bad copy / paste from the `ShapeRenderFilter`